templates/requirements.yml

```diff
@@ -356,7 +356,7 @@
   name: ssh
   activation_prefix: system_security_ssh_
 - src: git+https://github.com/Bergruebe/ansible-role-stirling-pdf.git
-  version: v0.29.0
+  version: v0.32.0
```

Comment:
There's no `v0.32.0` tag in https://github.com/Bergruebe/ansible-role-stirling-pdf, so this will not work.

Reply:
Alright, I also opened a PR there: https://github.com/Bergruebe/ansible-role-stirling-pdf/pull/2